sw/device/tests/i2c_target_test.c

```diff
@@ -124,11 +167,72 @@ static status_t write_transaction(ujson_t *uj, dif_i2c_t *i2c,
   return RESP_OK(ujson_serialize_i2c_transaction_t, uj, &txn);
 }
 
+static status_t enter_sleep(ujson_t *uj, dif_i2c_t *i2c, bool normal) {
+  dif_pinmux_wakeup_config_t wakeup_cfg = {
+      .mode = kDifPinmuxWakeupModeAnyEdge,
+      .signal_filter = false,
+      .pad_type = kDifPinmuxPadKindMio,
+      .pad_select = kTopEarlgreyPinmuxInselIoa7,
+  };
+  TRY(dif_pinmux_wakeup_detector_enable(&pinmux, 0, wakeup_cfg));
+  dif_pwrmgr_domain_config_t pwrmgr_domain_cfg = 0;
+  if (normal) {
+    // Normal sleep wakes up from an interrupt, so enable the relevant sources.
+    // Enable all the AON interrupts used in this test.
+    rv_plic_testutils_irq_range_enable(&plic, kTopEarlgreyPlicTargetIbex0,
+                                       kTopEarlgreyPlicIrqIdPwrmgrAonWakeup,
+                                       kTopEarlgreyPlicIrqIdPwrmgrAonWakeup);
+    // Enable pwrmgr interrupt
```

Comment:
Nit: For consistency 
```suggestion
    // Enable pwrmgr interrupt.
```

Reply:
Fixed.